src/MaterialComponents/Dialog/Error/index.test.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-undef */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import ErrorDialog from '.';
+
+describe('ErrorDialog', () => {
+  it('renders', () => {
+    const { container } = render(
+      <ErrorDialog
+        open
+        title="Error"
+        text="Something unexpected happened..."
+        confirmLabel="Close"
+        onClose={() => {}}
+      />,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Avoid to get an empty snapshot :
```suggestion
    expect(container).toMatchSnapshot();
```

Reply:
> Avoid to get an empty snapshot :

It doesn't really help, we just get an empty container instead. I think the problem comes from the mui animation, I'll leave a get around this later on.